models/order.js

```diff
@@ -21,13 +24,10 @@ OrderSchema.methods.sendSmsNotification = function(message, statusCallback, call
     statusCallback: statusCallback,
   };
 
-  client.messages.create(options)
-    .then((message) => console.log('Message sent to ' + message.customerPhoneNumber))
-    .catch((error) => console.log(error));
-
-  if (callback) {
-    callback.call(self);
-  }
+  return client.messages.create(options)
+    .then((message) => {
+      console.log('Message sent to ' + message.to);
+    })
```

Comment:
We added one but in routes/orders.js, there we can send a 500 response to the requester
https://github.com/TwilioDevEd/eta-notifications-node/blob/next-gen/routes/orders.js#L35